cnf/gac.in

```diff
@@ -114,7 +114,14 @@ c_link_dyn () {
     output_la=${1%.so}.la
     output_dir=$(dirname $1)
     ${c_dyn_linker} ${GAP_LDFLAGS} -o "$output_la" $2 ${c_addlibs} || exit 1
-    cp -r $output_dir/.libs/*.so $output_dir
+    if [ X"$SYS_IS_CYGWIN32" = X"yes" ] ; then
+        # GAP assumes shared libraries end in .so
+        for dllfile in `cd $output_dir/.libs; ls *.dll`; do
```

Comment:
This code is only run on cygwin, so why not  just `for dllfile in $output_dir/.libs/*.dll;` ? To guard against the possibility that no `.dll` was created? But that would be a serious bug, so in a sense, I'd prefer if it errored out here (even with a "strange" error), as opposed to running through and then leaving us scratching our head why a package DLL is missing, even though there was no visible error compiling it.

Reply:
The reason is because I want the name of the dll without `$output_dir/.libs`, so on the next line I can put it into `$output_dir`.

Reply:
So to avoid using `basename`? :-)

Anyway, if this works, fine by me.